test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/MapTest.scala

```diff
@@ -1032,4 +1133,6 @@ trait MapFactory {
   def allowsNullValuesQueries: Boolean = true
 
   def withSizeLimit: Option[Int] = None
+
+  def allowsIdentityBasedKeys: Boolean = false
```

Comment:
Consider renaming this to `isIdentityBased`, given that a) it's also for values and b) it's not *allowing*, it's *mandatory* (unlike `null` keys and values, for example).

Reply:
Definitely a better name, thanks.